### PR TITLE
[BE] [ADD] Resto view logic for Resto User - Eip 211

### PR DESCRIPTION
**Pull request by Neotehn**
### 🛠 Changes being made

Changes were made to the Resto-Site. User can now login via the frontend and see their assigned Restos

### CODE REVIEW

File: packages/backend/src/routes/restaurants.ts [hunk 1]
```diff
@@ -34,6 +37,30 @@ router.post('/', async (req, res) => {
     .send(restaurant);
 });
 
+
+router.get('/user/resto', async (req, res) => {
+  try {
+    const userID = await getUserIdResto(String(req.query.body));
```
Comment (JosefineM2000) [inline packages/backend/src/routes/restaurants.ts:43]: Can you add a comment that req.query.body contains the token so the required input is clearly stated here?

File: packages/restoWeb/src/pages/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -18,7 +18,8 @@ const HomePage = () => {
   }, []);
 
   const updateRestoData = () => {
-    getAllResto()
+    const userToken = localStorage.getItem('user');
+    getAllRestaurants(userToken)
```
Comment (JosefineM2000) [inline packages/restoWeb/src/pages/HomePage/HomePage.tsx:22]: Can we rename this to getAllRestaurantsByUser?

File: packages/backend/src/controllers/restaurantController.ts [hunk 3]
```diff
@@ -201,6 +238,7 @@ export async function createNewRestaurant(
   const upload = new RestaurantSchema({
     _id: id,
     name: obj.name,
+    userID: 0,
```
[the remark this answers is not in the document]
Comment (Neotehn) [reply packages/backend/src/controllers/restaurantController.ts:238]: Thats just the temporary default value. Will be modified in later tickets with the products/dishes/allergens

**Review by JosefineM2000 — COMMENTED** [review]
Please fix to show a message if not logged in to a message to log in / when no restos then show a message that no restos exist yet (similar to my dishes page) and make the add resto- button visible.

**Pull request merged**